Example Apps/Example Swift/ProxyManager.swift

```diff
@@ -173,11 +173,6 @@ extension ProxyManager: SDLManagerDelegate {
             vehicleDataManager.subscribeToVehicleOdometer()
         }
 
-        if newLevel == .full && firstHMILevelState != .full {
-            // This is our first time in a `FULL` state.
-            firstHMILevelState = newLevel
```

Comment:
The `firstHMILevelState` property needs to be removed, then re-test

Reply:
Looking at this again, won't we want to keep the `firstHMILevelState` property to recognize first time in a non-`NONE` state? Otherwise we will be sending commands and subscribing to vehicle data every time `newLevel` is non-`NONE` when `hmiLevel(oldLevel:didChangeToLevel:)` is called. Keeping this would also match the behavior of the Objective-C app.
https://github.com/smartdevicelink/sdl_ios/blob/e94851172e2b99408eb4a033a557438406d3cde1/Example%20Apps/Example%20Swift/ProxyManager.swift#L166-L174